UT4MasterServer/Models/Statistic.cs

```diff
@@ -24,4 +24,10 @@ public sealed class Statistic : StatisticBase
 
 	[BsonElement("window")]
 	public StatisticWindow Window { get; set; } = StatisticWindow.Daily;
+
+	[BsonElement("flagged"), BsonIgnoreIfNull]
+	public bool? Flagged { get; set; }
```

Comment:
Is there a reason for storing boolean?
`FlaggedProperties` can already have null which represents `Flagged` being false.

Reply:
I was actually thinking of that, the existance of **flaggedProperties** can be use to determine if statistic is flagged or not.
I will remove the boolean property and leave only list property, and rename it to **flagged**.